src/openloco/windows/TimePanel.cpp

```diff
@@ -330,39 +331,40 @@ namespace openloco::ui::TimePanel
     // 0x0043995C
     void formatChallenge()
```

Comment:
You should pass in the args from the tooltip function so that in the potential future we can remove the global var for common format args.

Reply:
Oh, good idea! This is also missing the `static` keyword, even though it has been declared as such, I notice.